static/src/stylesheets/module/commercial/_adverts-capi.scss

```diff
@@ -70,10 +72,49 @@
     }
 }
 
+.adverts--tone-supported {
+    background: none;
+
+    &::before {
+        content: '';
+        position: absolute;
+        top: 0;
+        left: 0;
+        right: 0;
+        border-top: 1px solid $news-main-2;
+
+        @include mq(tablet) {
+            left: calc(50% - 370px);
```

Comment:
Alas, no 😢  The top border should span the container's content box but there is no CSS to do that unfortunately, and since I'm using padding to center the content horizontally I have to make this small sacrifice.